rewatch/src/main.rs

```diff
@@ -111,3 +114,243 @@ fn get_lock(folder: &str) -> lock::Lock {
         acquired_lock => acquired_lock,
     }
 }
+
+fn parse_cli(raw_args: Vec<OsString>) -> Result<cli::Cli, clap::Error> {
```

Comment:
Makes me wonder if we can check if the first argument is any of the verbs, if not prepend the vector with `build`.
That way we avoid the double parse in the error case.